library/public/camera.h

```diff
@@ -49,6 +49,9 @@ class F3D_EXPORT camera
   virtual camera& setState(const camera_state_t& state) = 0;
   virtual camera_state_t getState() = 0;
   virtual void getState(camera_state_t& state) = 0;
+  virtual angle_deg_t getAzimuth() = 0;
+  virtual angle_deg_t getYaw() = 0;
+  virtual angle_deg_t getElevation() = 0;
```

Comment:
put that in another doxygen block and add some doc

Reply:
Added